src/scss/components/ec-card/_ec-card.scss

```diff
@@ -0,0 +1,8 @@
+.ec-card {
+  @include box-shadow-level-1;
+  @include shape-border-radius;
+
+  padding: 24px;
```

Comment:
The spec linked in PR says it should be 32px.

Reply:
Check it one more time, the image says in the card with the SVG that it have 32px of padding, but in the next, it says that now is 24px. I thought maybe the card with SVG have extra padding.

Let me ask Uve to confirm

Reply:
Uve confirm that, that should be with 24px and the CL 8px more